core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/MethodConfigApiServiceSpec.scala

```diff
@@ -365,10 +398,18 @@ class MethodConfigApiServiceSpec extends ApiServiceSpec {
       }
   }
 
-  it should "update the workspace last modified date on update method configuration" in withTestDataApiServices { services =>
+  it should "return 200 on put method configuration" in {
+    check200AddMC(Put)
+  }
+
+  it should "return 200 on post method configuration" in {
+    check200AddMC(Post)
+  }
```

Comment:
this was a really annoying pattern but it's the best i could do with FlatSpec. way too much work to extract the spec type so i could change this class to FreeSpec